src/modules/game/game.service.ts

```
@@ -139,9 +206,16 @@ export class GameService {
       });
     }
 
-    return game.save();
+    const savedGame = await game.save();
```

ESLint

[error] Insert `··`

(prettier/prettier)